onduty/types/index.d.ts

```diff
@@ -22,4 +22,13 @@ declare module 'onduty' {
     error(message: string, ...extraInfo: unknown[]): void;
     info(message: string, ...extraInfo: unknown[]): void;
   }
+
+  export interface Rotation {
+    primary: string;
+    secondary: null | string;
+  }
```

Comment:
GitHub teams don't have a concept for which members are primary or secondary, right? Could this be an array of usernames instead?

Reply:
This is the format of the payload from the rotation reporter tool :)